graylog2-web-interface/src/components/configurations/UserConfig.tsx

```diff
@@ -105,6 +105,8 @@ const UserConfig = () => {
             <dd>{viewConfig.enable_global_session_timeout ? 'Enabled' : 'Disabled'}</dd>
             <dt>Timeout interval:</dt>
             <dd>{viewConfig.enable_global_session_timeout ? viewConfig.global_session_timeout_interval : '-'}</dd>
+            <dt>Allow access token for external users:</dt>
+            <dd>{viewConfig.allow_access_token_for_external_user ? 'Enabled' : 'Disabled'}</dd>
```

Comment:
On the display, we can place a space before the value:
![image](https://github.com/user-attachments/assets/734f0b26-3c2a-4de3-aa05-c488e956db2e)

Reply:
Will do!